src/grandorgue/GOOrganController.cpp

```diff
@@ -280,6 +282,14 @@ void GOOrganController::ReadOrganFile(GOConfigReader &cfg) {
   for (unsigned i = 0; i < m_elementcreators.size(); i++)
     m_elementcreators[i]->Load(cfg);
 
+  p_OnStateButton = GetButtonControl(GOSetter::KEY_ON_STATE);
+
+  if (p_OnStateButton) {
+    // we do not want to send midi events on m_OnStateButton togeather with
```

Comment:
Suggestion:
// we do not want to send midi events _for p_OnStateButton together_ with